htdocs/compta/facture/card-rec.php

```diff
@@ -1186,6 +1187,7 @@
 		}
 
 		// Call Hook formConfirm
+		$formconfirm = '';
```

Comment:
You erase the formconfirm that is set few lines before

Reply:
I have fixed it in an other pull request

Reply:
So i close this PR